srcs/utils/error_exit.c

```diff
@@ -27,8 +27,6 @@ void	error_message(t_error code)
 	/* can you check the input:     > test       to see if you have leaks or a segfault ? */
 	else if (code == NO_CMD)
 		ft_putstr_fd("There is no command.\n", STDERR_FILENO);

```

Comment:
One last question, I don't think "no command" should be an error, at least in bash it doesn't give any error, or did you see it printing out an error at some point?